src/resources/outcomeLists.js

```diff
@@ -1,5 +1,5 @@
 import createResourceDescription from './util/createResourceDescription'
 
 export const outcomeLists = createResourceDescription('outcome_lists', {
-  includes: ['outcomes'],
+  includes: ['outcome_outcome_lists', 'outcomes'],
```

Comment:
Shouldn't it also be added for `outcomes`? I guess you're not using it for now but it would make sense to have it there anyway I think?